src/InteractiveWindow/VisualStudio/VsInteractiveWindowPane.cs

```diff
@@ -0,0 +1,142 @@
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using Microsoft.VisualStudio.ComponentModelHost;
+using Microsoft.VisualStudio.Editor;
+using Microsoft.VisualStudio.OLE.Interop;
+using Microsoft.VisualStudio.Shell;
+using Microsoft.VisualStudio.Shell.Interop;
+using Microsoft.VisualStudio.Text.Editor;
+using Microsoft.VisualStudio.TextManager.Interop;
+
+namespace Microsoft.VisualStudio.InteractiveWindow.Shell
+{
+    /// <summary>
+    /// The backing type for <see cref="VsInteractiveWindow"/>.  It subclasses <see cref="ToolWindowPane"/> so
+    /// that the instance handed out through the public API does not.
+    /// </summary>
+    internal sealed class VsInteractiveWindowPane : ToolWindowPane
+    {
+        // Keep in sync with Microsoft.VisualStudio.Editor.Implementation.EnableFindOptionDefinition.OptionName.
+        private const string EnableFindOptionName = "Enable Autonomous Find";
+
+        private readonly IComponentModel _componentModel;
+        private readonly IVsEditorAdaptersFactoryService _editorAdapters;
+        private readonly IInteractiveEvaluator _evaluator;
```

Comment:
I basically moved the code verbatim.  There are probably a few tiny style tweaks and I made `_evaluator` readonly.